inst/private/check_and_convert.m

```diff
@@ -46,7 +46,7 @@
       obj{i} = get_sym_from_python(x);
     elseif (py.isinstance(x, list_or_tuple))
       obj{i} = check_and_convert(x);
-    elseif (py.isinstance (x, py.str))
+    elseif (py.isinstance (x, py.type(py.str)))
```

Comment:
I can see that this works but it doesn't feel semantically correct to me (with my Python hat one).

How about `py.type(py.str(''))` or `py.type(py.str())` (better? if its well-defined).  That way, it would DTRT even if Pythonic were to distinguish (again?) between `py.str` and `py.str()`

Reply:
How about `isa (x, 'py.str')`?